README.md

```diff
@@ -95,11 +86,11 @@ Calls appInfo on the conductor with the provided id.
 
 ### `.cellData( role_id ) -> Promise<CellData>`
 
-Returns the cell data by role id for the respective DNA instance.
+(NOT IMPLEMENTED) Returns the cell data by role id for the respective DNA instance.
 
 ### `.stateDump() -> Promise<any>`
 
-Calls the state dump function on user's sourcechain of the respective DNA instance.
+(NOT IMPLEMENTED) Calls the state dump function on user's sourcechain of the respective DNA instance.
 
 ### `.signIn( opts? ) -> Promise<AgentInfo>`
```

Comment:
Note to self for when I make a docs PR: Change all of these occurences of `AgentInfo` to `AgentState`